components/body-map.js

```diff
@@ -149,8 +149,10 @@ export function BodyMap({
                   const borderStyle = isSelected
                     ? "border-solid border-2 border-brand"
                     : "";
-                  const textColor = isDisabled ? "text-slate-400" : "";
-                  const highlight = isHighlighted ? "bg-green-200" : "";
+                  const textColor = isDisabled
+                    ? "text-button-selected-disabled"
+                    : "";
```

Comment:
Another option is to use `disabled:text-button-selected-disabled` without a condition — it’s always a class of the button element whether it’s disabled or not. Tailwind CSS only uses classes starting with `disabled:` if the element is actually disabled.

Reply:
Thanks for the tips!